content/news/2023.pml

```diff
@@ -54,6 +54,28 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="support-rust">
+  <a href="#support-rust"><span class="tag-span"># </span></a>
+    15 November 2023: Support for Rust now available on seL4
+  </div>
+<div>
+  <p>
+  For the last year, Nick Spinale, funded by the seL4 Foundation, has been developing support for the programming language Rust on top of seL4.
```

Comment:
We could be more precise by replacing `on top of seL4` with `in seL4 userspace`.